vignettes/articles/crane.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Introduction to the {crane} package"
+author: Emily de la Rua
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## What is {crane}?
+
+The {crane} package was created to supplement the {gtsummary} package by providing additional functionality for creating clinical trial tables used specifically within the pharmaceutical industry. Since gtsummary was created as a very general framework to build tables, we want to avoid cluttering that package with pharma-specific functions that are only used by a small subset of {gtsummary} users. Instead, this pharma-specific functionality is added to the {crane} package for pharmaceutical data scientists to use. 
```

Comment:
Can we put each sentence on its own line? It makes it easier to maintain going forward.

Reply:
Updated